### PR TITLE
Ask for WGOR, WWCT and FPR in spe5

### PR DESCRIPTION
**Pull request by totto82**
Add some more output that are asked for in the spe5 paper and shown in the OPM paper

### CODE REVIEW

File: spe5/SPE5.BASE [hunk 1]
```diff
@@ -395,6 +395,21 @@ WWPT
   'PROD'
 /
 
+WGOR
+  'INJW'
```
Comment (joakim-hove) [inline spe5/SPE5.BASE]: You can just have:
```
WGOR
/
```

and

```
WWCT
/
```

to get `WGOR` and `WWCT` for all wells, if you wish to explicitly list the wells I see no reason to include the injector wells in `WGOR` and `WWCT`?

**Comment by alfbr** [comment]
This is a welcome improvement, feel free to self-merge when you have everything ready.

**Comment by totto82** [comment]
jenkins build this with downstreams update_data please

**Comment by totto82** [comment]
@akva2  How can I update the reference data with this PR?

**Comment by akva2** [comment]
like you did, i just had to add you to the trigger list in opm-data.

**Comment by totto82** [comment]
Then I will try again

**Comment by totto82** [comment]
jenkins build this with downstreams update_data please

**Comment by akva2** [comment]
failure is due to memory exhaustion on the server. building the ebos based apps eats approximately 4GB per app and too many ran in parallel.

**Comment by totto82** [comment]
jenkins build this with downstreams update_data please

**Comment by andlaus** [comment]
opm-data is not used for regression testing anymore. closing.

**Pull request closed without merging**